src/Microsoft.Azure.WebJobs.Host/Constants.cs

```diff
@@ -15,5 +15,6 @@ internal static class Constants
         public const string AzureWebsiteInstanceId = "WEBSITE_INSTANCE_ID";
         public const string AzureWebsiteContainerName = "CONTAINER_NAME";
         public const string DateTimeFormatString = "yyyy'-'MM'-'dd'T'HH':'mm':'ss'.'fffK";
+        public const string FunctionsHostingConfigSectionName = "FunctionsHostingConfig";
```

Comment:
We don't want to have "Functions" specific naming/apis in the SDK. Wondering if there's a simple way that the config can be transformed for scaler hosts such that the SDK code is just looking for SDK scoped app setting names. Can you provide an example of the current name/value for this config so I can suggest a translation?

Reply:
I just reused the name for hosting configuration in current [function host](https://github.com/Azure/azure-functions-host/blob/5a70c857d1575060d39f0af874d88f5572a29a61/src/WebJobs.Script/Config/FunctionsHostingConfigOptionsSetup.cs#L25).
For functions host scenario the configuration will be built in function host and the section name must be the same in function host and in web jobs sdk. We can come up with different name, what about "HostingConfig"?